lib/functions.php

```diff
@@ -113,7 +120,7 @@ function normalize(string $uri): string
 {
     $parts = parse($uri);
 
-    if (!empty($parts['path'])) {
+    if (isset($parts['path'])) {
```

Comment:
same as above?

```suggestion
    if ($parts['path'] !== null) {
```

Reply:
yes, I should be consistent - `isset` and `!== null` are essentially the same in these places.
I will adjust...

Reply:
Change coming...